packages/core/src/components/LinearGradient.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { StyleProp, ViewStyle } from "react-native";
+import { LinearGradient } from "expo-linear-gradient";
+import { extractStyles } from "../utilities";
+import {
+  createColorProp,
+  FORM_TYPES,
+  GROUPS,
+  PROP_TYPES,
+} from "@draftbit/types";
+
+type LinearGradientComponentProps = {
+  color1: string;
+  color2: string;
+  color3?: string;
+  startX: number;
+  startY: number;
+  endX: number;
+  endY: number;
+  children?: React.ReactNode;
+  style?: StyleProp<ViewStyle>;
+};
+
+const LinearGradientComponent = ({
+  children,
+  color1,
+  color2,
+  color3 = undefined,
+  startX = 0,
+  startY = 0,
+  endX = 100,
+  endY = 0,
+  style,
+}: LinearGradientComponentProps) => {
+  const { viewStyles } = extractStyles(style);
+  const colors = [color1, color2, color3].filter((color) => color) as string[];
+  const start = { x: startX / 100, y: startY / 100 };
+  const end = { x: endX / 100, y: endY / 100 };
+  return (
+    <LinearGradient colors={colors} start={start} end={end} style={viewStyles}>
+      {children}
+    </LinearGradient>
+  );
+};
+
+export default LinearGradientComponent;
+
+export const SEED_DATA = {
```

Comment:
Could you take this SEED_DATA object and move it over to `packages/core/src/mappings/LinearGradient.ts`? You can replace what's in that file already.

The mappings live in a separate place because we've run into some source map issues in the past where having these objects in here cause issues.

Reply:
Donezo (also Fugunzo).

Reply:
🙏